src/styles/_global-nav.scss

```diff
@@ -146,7 +146,7 @@ $btn-selected-blue: #004A95;
     display: flex;
     flex-direction: row;
     align-items: center;
-    min-height: 48px;
```

Comment:
We should probably keep both now that I think about it. We also want to make sure that the header is never under 48px.

Reply:
good call, done!